js/data/defaultSearchProviders.js

```diff
@@ -4,12 +4,10 @@ const defaultSearchProviders = [
     name: 'OB1',
     logo: '../imgs/ob1searchLogo.png',
     localLogo: '../imgs/ob1searchLogo.png',
-    search: 'https://search.ob1.io/search',
     listings: 'https://search.ob1.io/listings/search',
-    reports: 'https://search.ob1.io/reports',
-    torsearch: 'http://my7nrnmkscxr32zo.onion/search',
     torlistings: 'http://my7nrnmkscxr32zo.onion/listings/search',
-    locked: true,
+    vendors: 'https://search.ob1.io/profiles/search?type=vendor',
```

Comment:
The search endpoint uses "vendor" for the vendor search, and doesn't return a "search" endpoint any more, this just updates our code to match.